.env.development

```diff
@@ -3,9 +3,11 @@
 
 NODE_ENV='development'
 
+ADA_URL=''
```

Comment:
Please also add these variables to the .env file (so it has all the possible variables as a template)